lexicon.txt

```diff
@@ -0,0 +1,1317 @@
+aaaa
+aaaaaaa
+aal
+abuflen
+ack
+acked
+ackno
+acks
+acount
+acs
+addr
+addressalignedbeats
+addrstart
+addtogroup
+advertize
+afe
+ahb
+amba
+ap
+api
+apis
+april
+arg
+arp
+arppacket
+arpprocesspacket
+arprefreshcacheentry
+arps
+ascii
+asf
+atmel
+automaticpadcrcstrip
+autonegotiation
+aws
+backoff
+backofflimit
+backpressure
+bactive
+bafter
+barry
+baseaddress
+bbbbbbb
+bbefore
+bcloseaftersend
+bcloserequested
+bconnpassed
+bconnprepared
+bds
+berkeley
+besr
+bexpired
+bfinaccepted
+bfinacked
+bfinlast
+bfinrecv
+bfinsent
+bhasinit
+bisforrx
+bist
+bitmask
+bitscanreverse
+bl
+blowwater
+bmallocerror
+bmayconnect
+bmsschange
+bool
+boolean
+boutstanding
+bpassaccept
+bpassqueued
+bpdg
+breleaseaftersend
+breusesocket
+broadcastframesreception
+brxstopped
+bsd
+bsendfullsize
+bsendkeepalive
+btimestamps
+buf
+buffermanagement
+buffna
+busershutdown
+bwaitkeepalive
+bwinchange
+bwinscaling
+byt
+bytecount
+callbacklist
+calloc
+camen
+carriersense
+cbmc
+ccfg
+cchannel
+ce
+cerrorbuffer
+cgcr
+ch
+checkoptions
+checkoptionsinner
+checkoptionsouter
+checksum
+checksummode
+checksumoffload
+cli
+clk
+cmd
+co
+col
+com
+cond
+config
+configassert
+const
+controlbuffersize
+copyallframe
+coverity
+cpu
+cpus
+cr
+crc
+crs
+csn
+csr
+csv
+ctrl
+cwd
+da
+daif
+datalen
+datasheet
```

Comment:
let's change this to two words in the text and remove this.

Reply:
Removed this from lexicon.txt